servers/src/main/java/tachyon/master/lineage/meta/LineageStore.java

```diff
@@ -186,6 +160,21 @@ public synchronized Lineage getLineage(long lineageId) {
   }
 
   /**
+   * Gets the lineage that has the given output file.
+   *
+   * @param fileId the file id
+   * @return the lineage containing the output file
+   * @throws LineageDoesNotExistException
+   */
+  public synchronized Lineage getLineageOfOutputFile(long fileId)
+      throws LineageDoesNotExistException {
+    Lineage lineage = mOutputFileIndex.get(fileId);
+    LineageDoesNotExistException.check(lineage != null, ExceptionMessage.LINEAGE_DOES_NOT_EXIST,
```

Comment:
This is harder to read than
```
if (lineage != null) {
  throw new LineageDoesNotExistException(ExceptionMessage.LINEAGE_DOES_NOT_EXIST.getMessage(fileId));
}
```
Are there advantages to the `check` method besides saving a line or two?